problems/validation.go

```diff
@@ -27,6 +31,17 @@ func (problem ValidationProblem) TrimEmpty() Problem {
 	return problem
 }
 
+func (problem ValidationProblem) Error() string {
+	sb := strings.Builder{}
+	sb.WriteString(fmt.Sprintf("%s ", problem.Title))
+
+	for i, reason := range problem.Reasons {
+		sb.WriteString(fmt.Sprintf("reason(%d): %s; ", i, reason.Error()))
```

Comment:
I think we want to print the reason name and "reason" here. The intent of a ValidationReason is to be read with `name + " " + reason` The number is not relevant here (I think)

Reply:
ah, ok. I added the number mostly in order to get some kind of delimiter. This is better.